composer.json

```diff
@@ -32,7 +32,8 @@
         }
     },
     "require": {
-        "php": ">=5.5"
+        "php": ">=5.5",
+        "ext-mbstring": ">=5.5"
```

Comment:
Technically, mbstring is not required, as `use_mb_convert_encoding` is false by default. I'll have to leave the composer.json as it was.